src/components/checkboxes/template.njk

```diff
@@ -48,18 +48,30 @@
     {% set id = item.id if item.id else idPrefix + "-" + loop.index %}
     {% set name = item.name if item.name else params.name %}
     {% set conditionalId = "conditional-" + id %}
+    {% set hasHint = true if item.hint.text or item.hint.html %}
```

Comment:
We don't use the item prefix for anything else in this block (e.g. `name`, not `itemName`) – do we think this is different?

Reply:
Whatever we do, it'd be good to be consistent within this block – id, name and conditionalId _aren't_ prefixed. If we think it's useful to have that prefix, we should do it for all of them.

Reply:
we have `itemHintId` because we already have `hintId`, the whole group's hint

Reply:
In that case they should probably all be prefixed. It isn't clear from the context why `itemHintId` is named differently (to avoid clash with the group level hint) 🤔